### ISSUE TITLE
Show Users their TAPS on login.

### ISSUE THREAD

**Issue by brucestull** [brucestull/technology-and-perceptibility#73]
A User is presented with a list of their TAPs after they login.

**Comment by brucestull** [comment]
Encountered issues on page load:
`vue.js:634 [Vue warn]: Error in mounted hook: "TypeError: Cannot read properties of undefined (reading 'length')"`
```
mounted: function() {
    this.csrf_token = document.querySelector("input[name=csrfmiddlewaretoken]").value
    this.loadCurrentUser()
    this.displayUserTaps()
    // this.loadTaps()
}
```

```
displayUserTaps: function() {
    // "currentUser.taps_detail.0"
    this.clearUserTaps()
    this.clearTaps()
    console.log(`this.currentUser.username: `, this.currentUser.username)
    console.log(`this.currentUser.taps_detail: `, this.currentUser.taps_detail)
    console.log(`this.currentUser.taps_detail.length: `, this.currentUser.taps_detail.length)
    for (let i = 0; i < this.currentUser.taps_detail.length; i++) {
            this.userTaps.push(this.currentUser.taps_detail[i])
            this.userTaps[i].localBrowsableApiUrl = `/api/v1/taps/${this.userTaps[i].id}/`
            this.userTaps[i].localBrowsableApiUrlDescription = `Browsable API link to this TAP with id of ${this.userTaps[i].id}.`
        }
    console.log(`this.userTaps`, this.userTaps)
    this.clearCurrentTap()
    this.clearEditingCurrentTap()
},
```

Some sort of timing issue with Vue.js maybe?
